cmd/device-data-api/api.go

```diff
@@ -100,12 +94,12 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, dbs func() *d
 	vehicleAddr := common.HexToAddress(settings.VehicleNFTAddress)
 
 	// token based routes
-	vTokenV1.Get("/history", tk.OneOf(vehicleAddr, []int64{controllers.NonLocationData, controllers.AllTimeLocation}), cacheHandler, deviceDataController.GetHistoricalRawPermissioned)
-	vTokenV1.Get("/status", tk.OneOf(vehicleAddr, []int64{controllers.NonLocationData, controllers.CurrentLocation, controllers.AllTimeLocation}), cacheHandler, deviceDataController.GetVehicleStatus)
-	vTokenV1.Get("/status-raw", tk.OneOf(vehicleAddr, []int64{controllers.NonLocationData, controllers.CurrentLocation, controllers.AllTimeLocation}), cacheHandler, deviceDataController.GetVehicleStatusRaw)
+	vTokenV1.Get("/history", tk.OneOf(vehicleAddr, []int64{int64(privileges.NonLocationData), int64(privileges.AllTimeLocation)}), cacheHandler, deviceDataController.GetHistoricalRawPermissioned)
```

Comment:
Maybe it should've been `int64` in shared? Whoops.

Reply:
Opened a PR if we want to use int64 instead https://github.com/DIMO-Network/shared/pull/55